solidity-frontend/src/main/java/org/sonarsource/solidity/frontend/SolidityTokensInfo.java

```diff
@@ -0,0 +1,100 @@
+package org.sonarsource.solidity.frontend;
+
+import org.antlr.v4.runtime.Vocabulary;
+import org.antlr.v4.runtime.VocabularyImpl;
+
+public class SolidityTokensInfo {
+
+  public static final String[] ruleNames = {
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make this member "protected". ([squid:S2386](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3AS2386))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=13&issues=AWP4SA8t23Tx-xDbYxRg&open=AWP4SA8t23Tx-xDbYxRg)

[ISSUE_KEY]: AWP4SA8t23Tx-xDbYxRg